packages/config/tests/validate/snapshots/tests.js.md

```diff
@@ -426,7 +426,7 @@ Generated by [AVA](https://avajs.dev).
 > Snapshot 1
 
     `When resolving config file packages/config/tests/validate/fixtures/build_publish_parent_root/base/netlify.toml:␊
-    Configuration property "build.publish" "/external/path" must be inside the repository root directory.`
+    Configuration property "build.publish" "publish" must be inside the repository root directory.`
```

Comment:
not sure if this is correct in the snapshot 🤔

Reply:
This is correct. Below is the normalisation snippet

```js
// If we're outside the root directory, we're potentially accessing
// system directories that may vary from system to system, so we
// normalize them to /external/path.
if (relativePath.startsWith('..')) {
  return unixify(`${prefix}/external/path`)
}
```

If it had resolved to `/external/path` then that would mean the **publish** directory is outside repository root.

Reply:
so it was wrong previously? 🤔

Reply:
Good chance.